README.md

```diff
@@ -22,3 +22,13 @@ npm run lint
 
 ### Customize configuration
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+
+### Used API
+ Link to used API repository [repository with used API](https://github.com/monterail/monte-cinema)
+```
+- run docker 
+- in one terminal in uploaded directory use : make start 
+- parallel in second terminal use: docker-compose exec foreman bin/rails db:setup
+- [visit](http://localhost:3000)
+```
```

Comment:
Good way to document key information about the project 😄

Also - you might want to close previous PRs because there is a lot of unrelated changes here 😄